redis/src/main/scala/zio/redis/api/Keys.scala

```diff
@@ -4,7 +4,7 @@ import java.time.Instant
 
 import zio.duration._
 import zio.redis.Input._
-import zio.redis.Output._
+import zio.redis.Output.{ Tuple2Output, _ }
```

Comment:
```suggestion
import zio.redis.Output._
```
That should import also `Tuple2Output`